src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -128,6 +74,70 @@ export const writePackageTransform = (options: NgPackagrOptions) =>
     return graph;
   });
 
+async function copyAssets(
+  graph: BuildGraph,
+  entryPointNode: EntryPointNode,
+  ngPackageNode: PackageNode,
+): Promise<void> {
+  const ngPackage = ngPackageNode.data;
+
+  const pathsIgnored: string[] = [
+    '.gitkeep',
+    '**/.DS_Store',
+    '**/Thumbs.db',
+    '**/node_modules/**',
+    `${ngPackage.dest}/**`,
+  ];
+
+  const assets: AssetEntry[] = [];
+
+  for (const item of ngPackage.assets.concat('LICENSE', '**/README.md')) {
+    let asset: AssetEntry;
+    if (typeof item == 'object') {
+      asset = item;
+      asset.glob = path.join(ngPackage.src, asset.input, asset.glob);
+    } else {
+      const glob = path.join(ngPackage.src, item);
+      const isDir = await stat(glob)
+        .then(stats => stats.isDirectory())
+        .catch(() => false);
+      asset = { glob: isDir ? path.join(glob, '**/*') : glob, input: '.', output: '/' };
+    }
+
+    asset.input = path.join(ngPackage.src, asset.input);
+    asset.output = path.join(ngPackage.dest, asset.output);
+    asset.ignore = [...(asset.ignore ?? []), ...pathsIgnored].map(glob => path.join(asset.input, glob));
+
+    const isOutputValid = !path.relative(ngPackage.dest, asset.output).startsWith('..');
+    if (!isOutputValid) {
+      throw new Error('Cannot write assets to a location outside of the output path.');
+    }
+
+    assets.push(asset);
+  }
+
+  for (const asset of assets) {
+    const fileSrcPaths = await globFiles(asset.glob, {
+      cwd: asset.input,
```

Comment:
```suggestion
      cwd: path.join(ngPackage.src, asset.input),
```

Reply:
Yes this way is better, sorry for my previous unthinking replies. I have few experiences in submitting PRs :[